action.yaml

```
@@ -0,0 +1,54 @@
+name: Bootstrap
+description: Bootstrap workflow setup for organization repositories
+author: Incognia
+inputs:
+  install-packages:
+    description: Packages to install
+    default: ''
+
+  setup-go:
+    description: Use inloco/actions-setup-go
+    default: ''
+
+  setup-java:
+    description: Use inloco/actions-setup-java
+    default: ''
```

Replace empty defaults by optional inputs.

```suggestion
  install-packages:
    description: Packages to install
    required: false
  setup-go:
    description: Use inloco/actions-setup-go
    required: false
  setup-java:
    description: Use inloco/actions-setup-java
    required: false
```

---

But then users will have to specify all inputs...